.github/workflows/tests.yml

```diff
@@ -51,13 +48,13 @@ jobs:
         name: Maximize build space
         uses: easimon/maximize-build-space@master
         with:
-          root-reserve-mb: 35000  # for pip packages in /tmp
+          root-reserve-mb: 25000  # for pip packages in /tmp
```

Comment:
Looks like VM types have changed so root volume is about 10GB smaller ([source](https://github.com/easimon/maximize-build-space/issues/39#issuecomment-1935591779))